crux-metrics/src/crux/metrics/dropwizard/cloudwatch.clj

```diff
@@ -0,0 +1,36 @@
+(ns crux.metrics.dropwizard.cloudwatch
+  (:require crux.metrics.dropwizard)
+  (:import [io.github.azagniotov.metrics.reporter.cloudwatch CloudWatchReporter]
+           [java.io Closeable]
+           [software.amazon.awssdk.services.cloudwatch CloudWatchAsyncClient]
+           [software.amazon.awssdk.regions.providers DefaultAwsRegionProviderChain]
+           [software.amazon.awssdk.auth.credentials DefaultCredentialsProvider]
+           [java.util.concurrent TimeUnit]
+           [com.codahale.metrics MetricRegistry MetricFilter]))
+
+(defn ^CloudWatchReporter report
+  [^MetricRegistry reg {::keys [region dry-run? jvm-metrics?] :as args}]
+  (let [cwac (-> (CloudWatchAsyncClient/builder)
+                 (.region (.getRegion (new DefaultAwsRegionProviderChain)))
```

Comment:
you're not using the passed in region here?

Reply:
this looks like it has a [sensible default](https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/awscore/client/builder/AwsClientBuilder.html#region-software.amazon.awssdk.regions.Region-) anyway, so we shouldn't override it unless the user has explicitly provided one

Reply:
yep, that was a todo that slipped through the cracks